src/drive/mobile/ducks/mediaBackup/index.js

```diff
@@ -27,32 +27,35 @@ const getDirID = async path => {
   return _id
 }
 
-const canBackup = (force, getState) => {
+const canBackup = (isManualBackup, getState) => {
   const backupOnAnyNetworkType = !getState().mobile.settings.wifiOnly
 
   return (
-    force ||
-    (getState().mobile.settings.backupImages &&
-      (isWifi() || backupOnAnyNetworkType))
+    (isManualBackup || getState().mobile.settings.backupImages) &&
+    (isWifi() || backupOnAnyNetworkType)
```

Comment:
This is the part that really changes. Before, when the user ordered a manual back up, we didn't respect the network setting (ie. if there was no wifi, it would still back up). Now in order to do the backup, you need:

1. Triggered by the automatic setting or a manual command
2. A WiFi connection, or enable backups without WiFi